### PR TITLE
Minor fix to handle missing tuplesWritten count in SqoopExecution.archive.

### PR DESCRIPTION
**Pull request by rowandavies**

### CODE REVIEW

File: maestro-core/src/main/scala/au/com/cba/omnia/maestro/core/exec/SqoopExecution.scala [hunk 1]
```diff
@@ -280,7 +280,7 @@ object SqoopEx extends Sqoop {
         TypedPipe.from(TextLine(src))
           .writeExecution(CompressibleTypedTsv[String](dest))
           .getAndResetCounters
-          .map(_._2.get(StatKeys.tuplesWritten).get)
+          .map( _._2.get(StatKeys.tuplesWritten).getOrElse(0L) )
```
Comment (SamRoberts) [inline maestro-core/src/main/scala/au/com/cba/omnia/maestro/core/exec/SqoopExecution.scala]: we don't generally put spaces between function call brackets and their arguments

**Comment by SamRoberts** [comment]
nothing minor about that fix :)

The travis test for the commit failed. The failing test in question dies fail spuriously from time to time, due to a concurrency bug in one of the libraries we use. Still, it would be good to re-run the travis test to be sure.

Apart from that, and the minor syntactic comment above: :+1:

**Comment by rowandavies** [comment]
Travis test passed.  I fixed the spaces, but messed up and committed without them, so they ended up being a separate commit:  9e1fbc21232889bd66cf0989e39b33c6f8b828fc

**Comment by rowandavies** [comment]
Note that #281 seems similar.

**Pull request closed without merging**